### PR TITLE
Updates made for IWSLT 2022

### PR DESCRIPTION
**Pull request by snukky**
This PR adds changes that have been made for running the text-based human evaluation at IWSLT this year.

@AppraiseDev/core-team

### CODE REVIEW

File: Dashboard/models.py [hunk 1]
```diff
@@ -205,6 +205,7 @@
     'pus': 'Pashto (پښتو)',
     'lzh': 'Classical Chinese (文言文)',
     'chv': 'Chuvash (Чӑвашла)',
+    'aeb': 'Tunisian Arabic',
```
Comment (cfedermann) [inline Dashboard/models.py:208]: Should be listed as Arabic (Tunisian) for better ordering in lists. Also, would be good to add the name in Arabic script.

Non-blocking for now.

**Review by cfedermann — APPROVED** [review]
Thank you, @snukky 😀🙏

**Review by cfedermann — APPROVED** [review]
Thanks, @snukky

**Review by cfedermann — APPROVED** [review]

File: Dashboard/models.py (hunk 1 above)
Comment (snukky) [reply Dashboard/models.py]: Updated, thanks!

**Review by cfedermann — APPROVED** [review]
Thank you, @snukky 😎

**Pull request merged**